frontend/src/app/components/acceleration-timeline/acceleration-timeline-tooltip.component.html

```diff
@@ -45,7 +45,7 @@
         }
       </tr>
       <tr *ngIf="['accelerated', 'mined'].includes(accelerationInfo.status) && hasPoolsData()">
-        <td class="label" i18n="transaction.accelerated-by-pool|Accelerated by pool">Accelerated by</td>
```

Comment:
Note: I initially added a new i18n entry
`accelerated-by-pool|Accelerated by pool` 
and did not use 
`accelerated-by-hashrate|Accelerated to hashrate` 
because I wasn't sure that `Accelerated by` would translate correctly when referring to pools instead of a hashrate value.